index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+</head>
+<body>
+  <div class="grid-container">
+    <nav class="navbar">
+      <div class="hamburger small-screen">
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
+        <li class="nav-item"><a href="#">Sponsor</a></li>
+        <li class="nav-item"><a href="#">News</a></li>
+        <li class="nav-item " ><a href="#">Campaign</a></li>
+      </ul>
+    </nav>
+    <div class="home">
+      <header class="title">
+        <p class="tagline spacing">"MTN Nyegenyege welcomes all"</p>
+        <h2 class="page-title page-title-1 spacing">THE MTN NYEGENYEGE FESTIVAL 2022</h2>
+        <p class="introduction spacing">
+          Nyege Nyege is a collective in Kampala, Uganda that 
+          promotes outsider music, primarily electronic, by African artists. 
+          It was founded in 2013 by expats Arlen Dilsizian and Derek Debru. 
+          In addition to organizing ongoing parties and an artist residency
```

Comment:
Please ensure that there's at least 1, but no more than 1, `h1` tag in each page according to the project requirement.